cds/modules/deposit/links.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2016 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Links for record serialization."""
+
+from __future__ import absolute_import, print_function
+
+import invenio_deposit.links
+from flask import current_app, request
+from invenio_records_files.links import default_bucket_link_factory
+
+
+def deposit_links_factory(pid):
+    """Factory for record links generation."""
+    links = invenio_deposit.links.deposit_links_factory(pid)
+
+    bucket_link = default_bucket_link_factory(pid)
+    if bucket_link is not None:
+        links['bucket'] = bucket_link
+
+    links['html'] = current_app.config['DEPOSIT_UI_ENDPOINT'].format(
```

Comment:
this can be removed https://github.com/inveniosoftware/invenio-deposit/blob/master/invenio_deposit/links.py#L61-L66 :)